docs/mining/proof-of-stake.md

```diff
@@ -26,6 +26,30 @@ You will also need to [Obtain DCR](/getting-started/obtaining-dcr.md) if you hav
 
 ---
 
+## ** How Does It Work? ** ##
+
+Purchasing a ticket for PoS is quite simple (see below) but what happens to it after you buy it?
+A ticket will go through a few stages in its lifetime:
+
+1. You buy a ticket using your wallet. The rest of the process is the same for both solo and pool tickets.
+2. The ticket enters the 'mempool'. Only 20 tickets are mined into each block. This is where tickets wait to be mined.
+3. Tickets are mined into a block in order of their `ticketfee`. Note that the ticketfee is priced per KB. 
+A ticket transaction is about a third (solo) or half (pool) a KB so the actual fee you pay will be scaled accordingly.
+4. **A -** If your ticket is mined into a block, it will enter immature status for 256 blocks (about 20 hours). During
```

Comment:
In markdown, the number in incremented automatically.  This means since you have step 4 twice the numbers are not the same when rendered and when looking at the text.  You should either use 1. for all of them or have each one go up.